dune_client/client.py

```diff
@@ -128,9 +129,25 @@ def get_result_csv(self, job_id: str) -> ExecutionResultCSV:
         response.raise_for_status()
         return ExecutionResultCSV(data=BytesIO(response.content))
 
-    def get_latest_result(self, query: Query) -> ResultsResponse:
-        """GET latest results from Dune API for `query_id`"""
-        response_json = self._get(route=f"query/{query.query_id}/results")
+    def get_latest_result(self, query: Union[Query, str, int]) -> ResultsResponse:
```

Comment:
Can we update the doc string to describe better this Either or type we're using here.

Reply:
done